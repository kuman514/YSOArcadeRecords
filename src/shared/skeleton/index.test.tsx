import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';

import Skeleton from '.';

describe('Skeleton', () => {
  it('should match the snapshot', () => {
    const { container } = render(
      <div>
        <Skeleton width="100px" height="100px" borderRadius="20px" />
        <Skeleton width="50px" height="100px" borderRadius="40%" />
        <Skeleton width="100px" height="50px" borderRadius="10rem" />
        <Skeleton width="10rem" height="4rem" borderRadius="1rem" />
        <Skeleton width="100px" height="100px" borderRadius="0" />
      </div>
    );
    expect(container).toMatchSnapshot();
  });
});
