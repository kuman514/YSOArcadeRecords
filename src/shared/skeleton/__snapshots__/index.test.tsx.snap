// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Skeleton > should match the snapshot 1`] = `
<div>
  <div>
    <div
      class="animate-pulse bg-slate-200"
      style="width: 100px; height: 100px; border-radius: 20px;"
    />
    <div
      class="animate-pulse bg-slate-200"
      style="width: 50px; height: 100px; border-radius: 40%;"
    />
    <div
      class="animate-pulse bg-slate-200"
      style="width: 100px; height: 50px; border-radius: 10rem;"
    />
    <div
      class="animate-pulse bg-slate-200"
      style="width: 10rem; height: 4rem; border-radius: 1rem;"
    />
    <div
      class="animate-pulse bg-slate-200"
      style="width: 100px; height: 100px; border-radius: 0;"
    />
  </div>
</div>
`;
